pkgs/development/python-modules/APScheduler/default.nix

```diff
@@ -47,6 +48,7 @@ buildPythonPackage rec {
     pytz
     tzlocal
     funcsigs
+    setuptools
```

Comment:
Please add a pythonImportsCheck to prevent breakages in the future.

Reply:
Weirdly enough this import check doesn't catch the bug:

```
pythonImportsCheck = [ "apscheduler" "pkg_resources" ];
```

Without the setuptools propagatedBuildInput, I get the following error when using APScheduler. It seems the import error should be tested with that `pythonImportsCheck` right?
```
Mar 06 20:28:25 acturia bot.py[11305]:   File "/nix/store/jr04ipzkmy9kn1kjrbqvybm92j4qirr3-python3.8-python-telegram-bot-13.3/lib/python3.8/site-packages/telegram/ext/jobqueue.py", line 27, in <module>
acturia> Mar 06 20:28:25 acturia bot.py[11305]:     from apscheduler.events import EVENT_JOB_ERROR, EVENT_JOB_EXECUTED, JobEvent
Mar 06 20:28:25 acturia bot.py[11305]:   File "/nix/store/lgv34l1ilb1hd4qkjm9gm71ld7rxanqs-python3.8-APScheduler-3.7.0/lib/python3.8/site-packages/apscheduler/__init__.py", line 1, in <module>
Mar 06 20:28:25 acturia bot.py[11305]:     from pkg_resources import get_distribution, DistributionNotFound
Mar 06 20:28:25 acturia bot.py[11305]: ModuleNotFoundError: No module named 'pkg_resources'
```

Reply:
It should be ``pythonImportsCheck = [ "apscheduler" ];``

Reply:
Added the imports check